keyboards/checkerboards/snop60/keyboard.json

```
@@ -47,6 +47,16 @@
     },
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
+    "features": {
```

Could mitigate the oversize issue?
```suggestion
    "build": {
        "lto": true
    }
    "features": {
```